store/game_read.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../classes/game'
+require 'json'
+
+def read_games
+  data = []
+  if File.exist?('games.json')
```

Comment:
You forgot to update this path :)
```suggestion
  if File.exist?('data/games.json')
```

Reply:
Hi  @eduardosancho ,
oh, my bad. thank you for your review.